Tools/CMake/AMReX_Config.cmake

```
@@ -154,11 +154,13 @@ function (configure_amrex)
         cmake_policy(GET CMP0105 host_link_supported)
       endif()
 
-      if(APPLE)
+      if(CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang")
         if( host_link_supported STREQUAL "NEW" ) # CMake 3.18+
-          target_link_options(amrex PUBLIC "LINKER:-undefined,warning")
+          target_link_options(amrex PUBLIC "LINKER:-U,_amrex_probinit")
+          target_link_options(amrex PUBLIC "LINKER:-U,_getLevelBld")
```

Mangle like a robot:
```suggestion
          target_link_options(amrex PUBLIC "LINKER:-U,_Z11getLevelBldv")
```
https://en.wikipedia.org/wiki/Name_mangling#How_different_compilers_mangle_the_same_functions